src/Database/QueryCompiler.php

```diff
@@ -85,6 +85,13 @@ class QueryCompiler
      */
     protected $_orderedUnion = true;
 
+    /**
+     * Indicate whether aliases in SELECT clause need to be always quoted.
+     *
+     * @var bool
+     */
+    protected $_quotedSelectAliases = false;
```

Comment:
I'm half tempted to not have protected property overrides and instead duplicate `_buildSelectPart` into the Postgres compiler so that both the postgres and non-postgres versions have no additional branches. Our test suite is thorough enough that we should be able to keep changes aligned between the two implementations.

Reply:
I too had contemplated the same thing :)

I will update accordingly.

Reply:
@markstory Duplicating `_buildSelectPart()` will only avoid a `if` check in `PostgresCompiler`. The `if` check will still be required in `QueryCompiler` if auto quoting is enabled.

Reply:
Well that doesn't seem worth it.